packages/rolldown/tests/fixtures/plugin/generate-bundle/_config.ts

```diff
@@ -31,13 +31,16 @@ export default defineTest({
           expect(isWrite).toBe(true)
           // Mutate chunk
           chunk.code = 'console.error()'
+          // Delete chunk
+          delete bundle['index.js']
```

Comment:
How rollup deletes assets? Are we introducing new API?

Reply:
> How rollup deletes assets? Are we introducing new API?
Rollup just using `detele`, the new API is unnessary.